ts/Core/Series/SeriesDefaults.ts

```diff
@@ -451,6 +451,9 @@ const seriesDefaults: SeriesOptions = {
     /**
      * The line cap used for line ends and line joins on the graph.
      *
+     * @sample {highcharts|highstock} highcharts/series/linecap/
```

Comment:
Is it necessary? Could you generate local docs and see if it's working the same way without it?
```{highcharts|highstock}```

Reply:
You're right! It's not necessary, everything is working fine without `{highcharts|highstock}`.